packages/run-protocol/src/voting.js

```diff
@@ -0,0 +1,64 @@
+// @ts-check
+
+import '@agoric/governance/src/exported.js';
+import '@agoric/zoe/exported.js';
+import '@agoric/zoe/src/contracts/exported.js';
+
+import { E } from '@endo/eventual-send';
+
+/**
+ * @file
+ *
+ * This extremely quick and dirty contract simplifies the process of
+ * calling for votes on changes to the economy by gathering the governor creator
+ * facets in one place and invokes `voteOnParamChanges` and
+ * `voteOnApiInvocation` on each when requested. A cleaner version would
+ * validate parameters, constrain deadlines and probably split the ability to
+ * call for vote into separate capabilities for finer grain encapsulation.
+ */
+
+/**
+ * @param {ZCF<{binaryVoteCounterInstallation:Installation}>} zcf
+ * @param {{reserve:GovernedContractFacetAccess<unknown>, amm:GovernedCreatorFacet<unknown>, vaults:GovernedCreatorFacet<unknown>}} privateArgs
+ */
+export const start = async (zcf, privateArgs) => {
+  const { binaryVoteCounterInstallation: counter } = zcf.getTerms();
+  const { reserve, amm, vaults } = privateArgs;
+
+  const voteOnParamChanges = (
+    target,
+    params,
+    path = { paramPath: { key: 'governedApi' } },
+    deadline,
+  ) => {
+    return E(target).voteOnParamChanges(counter, deadline, {
+      paramPath: path,
+      changes: params,
+    });
+  };
+
+  const voteOnApiInvocation = (contract, method, amounts, deadline) => {
+    return E(reserve).voteOnApiInvocation(method, amounts, counter, deadline);
```

Comment:
```suggestion
    return E(contract).voteOnApiInvocation(method, amounts, counter, deadline);
```

Reply:
That was intentional, since reserve is the only one with an ApiInvocation, but we can make it general.

Reply:
or take out the `contract` arg